src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -173,41 +153,38 @@ Before starting our [guided install](https://one.newrelic.com/nr1-core?state=51f
 
 <InstallFeedback />
 
-## Upgrading our Kubernetes integration [#update]
+### Use guided install [#installer]
 
-Our Kubernetes integration is under active development and we regularly release updates which include bug fixes, new features, and support for newer Kubernetes versions and cloud providers. We strongly recommend all our customers to regularly update the Kubernetes integration to get the best experience.
+We encourage you to use our guided install for servers, VMs, and unprivileged environments. The guided install can provide you either a Helm command with the required values filled, or a plain manifest if you don't want to use Helm. It also offers control over which features are enabled and which data is collected. If you are installing our integration on a managed cloud, please take a look at these [preliminary notes](#cloud-platforms) before proceeding.
 
-### Upgrading using the guided install and Helm [#update-helm]
+Before you start the guided install, make sure you have a New Relic account. It's free and no credit card is required.
 
-In order to update an installation that was deployed using Helm command provided by the [guided install](https://one.newrelic.com/nr1-core?state=51fbbd48-c8ca-ead9-bb90-af96e18d82a7), just go through the process and run the Helm command again. This will pull the new version of the chart and its dependencies and upgrade it to the latest version.
+If your New Relic account [is in the EU region](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), choose the EU install option.
```

Comment:
@nbaenam I'm not sure if this is quite right. I don't see an EU option in the UI. Rather, I think users need to start from the EU link: [one.eu.newrelic.com](http://one.eu.newrelic.com/launcher/k8s-cluster-explorer-nerdlet.cluster-explorer-launcher?pane=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJndWlkZWQiLCJlbnYiOiJrdWJlcm5ldGVzIiwiaW5pdGlhbEFjdGlvbkluZGV4IjpudWxsLCJhY3Rpb25JbmRleCI6MX0=)

Could we add a third button for the EU?